HenHen.Framework/Graphics2d/Sprite.cs

```diff
@@ -5,43 +5,21 @@ namespace HenHen.Framework.Graphics2d
 {
     public class Sprite : Drawable
     {
-        public ColorInfo Color { get; set; } = new ColorInfo(255, 255, 255);
         private Raylib_cs.Texture2D texture;
-        private bool isTextureLoaded;
+
+        public ColorInfo Color { get; set; } = new ColorInfo(255, 255, 255);
 
         /// <summary>
         /// Loads a texture from a file path. Previous texture is automatically unloaded.
         /// </summary>
         /// <param name="path"></param>
-        public void SetTexture(string path)
-        {
-            UnloadTexture(texture);
-            texture = Raylib_cs.Raylib.LoadTexture(path);
-            isTextureLoaded = true;
-        }
```

Comment:
This function should remain - use Game.TextureStore here to set the `texture` field.